app/views/shared/_image_shares_modal.erb

```diff
@@ -0,0 +1,26 @@
+<div class="modal fade" id="imageSharesModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
```

Comment:
Can we update `aria-labelledby` attribute to make more sense?

Reply:
Minor: camel case is not the pattern defining an `id`, do you guys wanna change it to `image-share-modal` instead?

Reply:
Sure